package.json

```diff
@@ -38,14 +39,16 @@
     "**/src/**/*.js": [
       "prettier --write",
       "eslint --fix",
-      "git add"
+      "git add",
+      "test"
```

Comment:
This should be `test:staged` instead of just `test`, right?
Also, we should have test script before `git add`, so that we make sure that `git add` only happens if the tests passed successfully.